config/uwsgi.ini

```diff
@@ -26,14 +26,16 @@ cheaper-overload = 1
 cheaper = 50
 
 # Max Workers
-workers = 200
+workers = 100
```

Comment:
The plan was to keep this at 200 and monitor closely. With the limits in place too, uwsgi hopefully keeps the instance from falling over due to memory, but still allows near 200 workers if memory is okay.